.clippy.toml

```diff
@@ -0,0 +1 @@
+msrv = "1.30.0"
```

Comment:
Well this was pleasant surprise. But isn't it a bit low?

Reply:
Yep, looks suspiciously low.

Reply:
Oops! 😅 My mistake, copy-pasted without changing.